contracts/src/v0.8/vrf/dev/VRFCoordinatorV2_5.sol

```diff
@@ -444,6 +453,32 @@ contract VRFCoordinatorV2_5 is VRF, SubscriptionAPI, IVRFCoordinatorV2Plus {
     bool onlyPremium
   ) external nonReentrant returns (uint96 payment) {
     uint256 startGas = gasleft();
+    // fulfillRandomWords msg.data has 772 bytes and with an additional
```

Comment:
Wait, is this code from the other [PR](https://github.com/smartcontractkit/chainlink/pull/12325/files)? Did this end up here by accident or on purpose?

Reply:
I based this PR on the previous PR :)
Anyways, we can merge only one by one right